src/LanguageServer/Impl/Implementation/Server.Documents.cs

```diff
@@ -56,9 +56,13 @@ public sealed partial class Server {
         }
 
         public void DidChangeWatchedFiles(DidChangeWatchedFilesParams @params) {
+            _disposableBag.ThrowIfDisposed();
             foreach (var c in @params.changes.MaybeEnumerate()) {
-                _disposableBag.ThrowIfDisposed();
-                // TODO: handle?
+                switch (c.type) {
+                    case FileChangeType.Deleted:
```

Comment:
Do you know if this case gets hit?

![image](https://user-images.githubusercontent.com/5341706/54242863-68eae400-44e3-11e9-8cc6-1f192e1582a6.png)

`b` and `food` still seem to exist after I make `b` with `def foo(): return 123`, import it in `a`, then delete `b`, but I could be testing something different than what this handles.

Reply:
Frankly, I haven't seen it hit. I don't know when VSC sends it, this is why we have manual library watcher. But it doesn't hurt to handle.

Reply:
We could start watching workspace manually too, but that's a different issue.